dev/create-release/release-build.sh

```diff
@@ -322,18 +322,18 @@ if [[ "$1" == "package" ]]; then
   #   'python/pyspark/install.py' and 'python/docs/source/getting_started/install.rst'
```

Comment:
you missed this comment..

Reply:
> you missed this comment..

I know this, but since the package is not correct, change `install.py` will failed since there's no related package on it. Right?

Reply:
It has to be renamed to hadoop-3, and should fix the documentation at https://spark.apache.org/docs/latest/api/python/getting_started/install.html#using-pypi accordingly. Let's don't make divergence on profile name alone. This kind of change requires to change infra in all forks, other companies, and how we release. I would like to avoid make incremental changes on this kind of item.

Reply:
So we can make `PYSPARK_HADOOP_VERSION ` as 3 but still install spark-3.2.0-bin-hadoop3.2?

Reply:
Once 3.3 is released, it should be `PYSPARK_HADOOP_VERSION=3`. We should document that. This part I can help.